cishouseholds/pipeline/survey_responses_version_2_ETL.py

```diff
@@ -394,6 +390,63 @@ def union_dependent_transformations(df):
         visit_status_column="participant_visit_status",
         visit_date_column="visit_datetime",
     )
+    df = update_column_values_from_map(
```

Comment:
Please move this to transform_survey_responses_version_2_delta, so that it's only applied to V2 data